linkage-monitor/src/main/java/com/google/cloud/tools/dependencies/linkagemonitor/LinkageMonitor.java

```diff
@@ -105,10 +105,20 @@ public static void main(String[] arguments)
       RepositorySystem repositorySystem, RepositorySystemSession session, Path projectDirectory) {
     ImmutableMap.Builder<String, String> artifactToVersion = ImmutableMap.builder();
     Iterable<Path> paths = MoreFiles.fileTraverser().breadthFirst(projectDirectory);
+
+    PATHS_LOOP:
     for (Path path : paths) {
       if (!path.getFileName().endsWith("pom.xml")) {
         continue;
       }
+      for (Path pathElement : path) {
+        String name = pathElement.toString();
+        if (name.equals("build") || name.equals("target")) {
+          // Exclude Gradle's build directory and Maven's target directory.
+          continue PATHS_LOOP;
```

Comment:
Is this truly necessary? Usually there's a cleaner approach that does not use continue. Here I think you can reverse the if and avoid continue.

Reply:
Removed the label. Does this look good now?